website/content/docs/secrets/transform/tokenization.mdx

```diff
@@ -25,6 +25,38 @@ Depending on the mapping mode, the plaintext may be decoded only with possession
 of the distributed token, or may be recoverable in the export operation. See
 [Security Considerations](#security-considerations) for more.
 
+### Convergence
+
+By default, tokenization produces a unique token for every encode operation. 
+This makes the resulting token fully independent of its plaintext and expiration. 
```

Comment:
It may be unclear what _this_ refers to. 

Suggestion, along these lines:
Producing a unique token results in the token being fully independent of its plaintext and expiration.